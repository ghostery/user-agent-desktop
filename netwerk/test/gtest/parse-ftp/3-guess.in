I couldn't find any SuperTCP or Chameleon/Newt FTP servers, so the
following is based on guesswork from the mirror.pl and lynx projects.
The first 8 are Chameleon/Newt, the remaining 16 entries are SuperTCP
with two different date styles. The precise amount of whitespace 
between columns might not be correct, but the parser shouldn't care.

.               <DIR>      Nov 16 1994 17:16   
..              <DIR>      Nov 16 1994 17:16   
INSTALL         <DIR>      Nov 16 1994 17:17
CMT             <DIR>      Nov 21 1994 10:17
DESIGN1.DOC     11264      May 11 1995 14:20   A
README.TXT       1045      May 10 1995 11:01
WPKIT1.EXE     960338      Jun 21 1995 17:01   R
CMT.CSV             0      Jul 06 1995 14:56   RHA
.               <DIR>           11-16-94        17:16
..              <DIR>           11-16-94        17:16
INSTALL         <DIR>           11-16-94        17:17
CMT             <DIR>           11-21-94        10:17
DESIGN1.DOC          11264      05-11-95        14:20
README.TXT            1045      05-10-95        11:01
WPKIT1.EXE          960338      06-21-95        17:01
CMT.CSV                  0      07-06-95        14:56
.               <DIR>           11/16/94        17:16
..              <DIR>           11/16/94        17:16
INSTALL         <DIR>           11/16/94        17:17
CMT             <DIR>           11/21/94        10:17
DESIGN1.DOC          11264      05/11/95        14:20
README.TXT            1045      05/10/95        11:01
WPKIT1.EXE          960338      06/21/95        17:01
CMT.CSV                  0      07/06/95        14:56
