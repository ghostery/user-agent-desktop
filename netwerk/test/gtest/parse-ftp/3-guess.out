<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->

11-16-1994  17:16:00       <DIR>          .
11-16-1994  17:16:00       <DIR>          ..
11-16-1994  17:17:00       <DIR>          INSTALL
11-21-1994  10:17:00       <DIR>          CMT
05-11-1995  14:20:00                11264 DESIGN1.DOC
05-10-1995  11:01:00                 1045 README.TXT
06-21-1995  17:01:00               960338 WPKIT1.EXE
07-06-1995  14:56:00                    0 CMT.CSV
11-16-1994  17:16:00       <DIR>          .
11-16-1994  17:16:00       <DIR>          ..
11-16-1994  17:17:00       <DIR>          INSTALL
11-21-1994  10:17:00       <DIR>          CMT
05-11-1995  14:20:00                11264 DESIGN1.DOC
05-10-1995  11:01:00                 1045 README.TXT
06-21-1995  17:01:00               960338 WPKIT1.EXE
07-06-1995  14:56:00                    0 CMT.CSV
11-16-1994  17:16:00       <DIR>          .
11-16-1994  17:16:00       <DIR>          ..
11-16-1994  17:17:00       <DIR>          INSTALL
11-21-1994  10:17:00       <DIR>          CMT
05-11-1995  14:20:00                11264 DESIGN1.DOC
05-10-1995  11:01:00                 1045 README.TXT
06-21-1995  17:01:00               960338 WPKIT1.EXE
07-06-1995  14:56:00                    0 CMT.CSV
